DependencyInjection/CompilerPass/PluginPass.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Http\HttplugBundle\DependencyInjection\CompilerPass;
+
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+/**
+ * Remove plugins if their dependencies are missing.
+ *
+ * @author Tobias Nyholm <tobias.nyholm@gmail.com>
+ */
+class PluginPass implements CompilerPassInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function process(ContainerBuilder $container)
+    {
+        if (!($container->hasDefinition('logger') || $container->hasAlias('logger'))) {
```

Comment:
this is not really correct. its possible to configure a different logger, at which point this is no longer needed. i think we should use a container parameter to know which logger service must exist.